README.md

```diff
@@ -1,5 +1,7 @@
 # Packaging for fluent-bit
 
+[![dockerhub](https://img.shields.io/docker/v/rancher/fluent-bit?sort=semver&style=flat-square)](https://hub.docker.com/r/rancher/fluent-bit/tags)
```

Comment:
Note: this is unrelated, but helpful for users who want to go straight to the Dockerhub page.